components/datatypes/mediaevent.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/mediaEvent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Media event information",
+  "type": "object",
+  "meta:status": "experimental",
+  "meta:extensible": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
```

Comment:
Adding this behavior will turn this into a class. I guess you just want to create a data type here right ? Please clarify.

Reply:
yes, we need the mediaEvent datatype to have the format of e.g.:
```
{
      "eventType": "media.adBreakStart",
      "timestamp": "2022-04-13T15:51:05+03:00",
      "mediaCollection": {
        "playhead": 548,
        "sessionDetails": {
          "ID": "someID"
        },
        "advertisingPodDetails": {
          "friendlyName": "mid-roll",
          "index": 1,
          "second": 548
        }
}
```
so the timestamp and eventType from timeseries would be just as simple fields, not inside another datatype, if this is not possibile using `extends` one option would also be to add these fields manually.